src/App.js

```diff
@@ -4,16 +4,18 @@ import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import MainPage from "./components/Main";
 import Navbar from "./components/Navbar";
 import Footer from "./components/Footer";
+import Card from "./components/carte-description/carte-description";
 
 function App() {
   return (
     <Router>
       <div>
         <Navbar />
-        <div className="container"></div>
         <Switch>
           <Route exact path="/" component={MainPage} />
+          <Route path="/:name" component={Card} />
```

Comment:
I'd recommend `/card/:name`, to avoid some edge cases.
Example: A card could be named "Booster" and accidentally lead to the `/booster` route meant to open a random booster ;-)

Reply:
done